### PR TITLE
Add http-parser package

### PR DESCRIPTION
**Pull request by isaachier**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/isaachier/hunter/build/1.0.384
  * https://travis-ci.org/isaachier/hunter/builds/352623945

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/209

### CODE REVIEW

File: cmake/projects/http-parser/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME http-parser
+    VERSION "2.8.0"
+    URL "https://github.com/isaachier/http-parser/archive/hunter-2.8.0-p0.tar.gz"
```
Comment (ruslo) [inline cmake/projects/http-parser/hunter.cmake]: Please use https://github.com/hunter-packages/http-parser here.

**Comment by ruslo** [comment]
> Here is the links to the Travis/AppVeyor with status "All passed"

I got "Oops, we couldn't find that build!" message.

**Comment by isaachier** [comment]
Sorry I had AppVeyor and Travis build numbers reversed. You should be able to see it now that I've corrected the links.

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3451
* https://travis-ci.org/ingenue/hunter/builds/353200591?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3453
* https://travis-ci.org/ingenue/hunter/builds/353209286?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.20.19

**Pull request merged**